provider/lib/lib.go

```diff
@@ -41,4 +47,5 @@ func GetConfig(filepath string) ([]Config, error) {
 type ProviderEvaluateResponse struct {
 	Passed              bool
 	ConditionHitContext []map[string]string
+	Filepaths           []string
```

Comment:
Thoughts on this? I figured a list of filepaths would be the most common result needed for pipelining so went ahead and made it top-level

Reply:
I feel like we need to design this with an API, but I think you make a good point, I would also want though, like 

`map[<filepath>][]lineNumbers`

IDK

Reply:
I've removed this